core/src/test/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -6186,6 +6186,31 @@ public void subTestIntervalDayFailsValidation() {
     assertEquals(linux(sqlNodeVisited1.toString()), str1);
   }
 
+  @Test void testSqlDeleteSqlBasicCallToString() throws Exception {
+    final String sql0 = "delete from emps where empno = 1";
+    final SqlNode sqlNode0 = getSqlParser(sql0).parseStmt();
+    final SqlNode sqlNodeVisited0 = sqlNode0.accept(new SqlShuttle() {
+      @Override public SqlNode visit(SqlIdentifier identifier) {
+        return new SqlIdentifier(identifier.names,
+            identifier.getParserPosition());
+      }
+    });
+    final String str0 = "DELETE FROM `EMPS`\n"
+        + "WHERE `EMPNO` = 1";
+    assertEquals(linux(sqlNodeVisited0.toString()), str0);
```

Comment:
we don't use assertEquals. because people get the args in the wrong order. as you did.